README.md

```diff
@@ -1,45 +1,55 @@
-# XRP Wallet App for Ledger Nano S and Ledger Nano X
+# XRP Wallet App for Ledger devices
+
+[![Ensure compliance with Ledger guidelines](https://github.com/LedgerHQ/app-flow/actions/workflows/guidelines_enforcer.yml/badge.svg)](https://github.com/LedgerHQ/app-flow/actions/workflows/guidelines_enforcer.yml)
```

Comment:
Badges should point to /app-xrp/ instead of /app-flow/

Reply:
Can you open a PR to fix it ? I'll do it otherwise

Reply:
will do